packages/protocol/test-sol/stability/SortedOracles.t.sol

```diff
@@ -0,0 +1,592 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.5.13;
+pragma experimental ABIEncoderV2;
+
+import { Test } from "celo-foundry/Test.sol";
+import { SortedOracles } from "../../contracts/stability/SortedOracles.sol";
+import "../../contracts/common/FixidityLib.sol";
+import "../../contracts/common/linkedlists/AddressSortedLinkedListWithMedian.sol";
+import "../../contracts/common/linkedlists/SortedLinkedListWithMedian.sol";
+import { Constants } from "../constants.sol";
+
+contract SortedOraclesTest is Test, Constants {
+  using FixidityLib for FixidityLib.Fraction;
+  using AddressSortedLinkedListWithMedian for SortedLinkedListWithMedian.List;
+
+  SortedOracles sortedOracle;
+
+  address oracleAccount;
+  address aToken = 0x00000000000000000000000000000000DeaDBeef;
+  uint256 reportExpiry = 1 * 60 * 60; // 1 hour
```

Comment:
NIT: use constants

Reply:
we'd need to cherry pick the constants contract from master, not sure it's the right time for that.

Reply:
I added it to constants